config/babel-register/babel-hooks.mjs

```diff
@@ -87,7 +96,14 @@ export async function resolve(specifier, context, nextResolve) {
  * the last user-supplied load hook
  */
 export async function load(url, context, nextLoad) {
-    const { format, shortCircuit, source } = await nextLoad(url, context);
+    const { format, shortCircuit, source } = await nextLoad(url, context)
+        .catch(async (error) => {
+            if (error.code === 'ERR_UNKNOWN_FILE_EXTENSION') {
```

Comment:
Can you comment why you need this specific case?

Reply:
The default loader can load ESM and CJS '.js' files, however '.ts' files shall be loaded with `format: module`.

Reply:
For more information, `mocha` is loaded as an `CJS` module so we can't directly load with the `format: module`.